protocol/lib/metrics/metric_keys.go

```diff
@@ -0,0 +1,39 @@
+package metrics
+
+// Metrics Keys Guidelines
+// 1. Be wary of length
+// 2. Prefix by module
+// 3. Suffix keys with a unit of measurement
+// 4. Delimit with '_'
+// 5. Information such as callback type should be added as tags, not in key names.
```

Comment:
This is helpful, and I would recommend extending a bit further:
1. Provide examples here (for ex. what is a reasonable length, what is an ex. module name (`x/clob` or `clob`), etc.
2. Would be helpful to formalize this into a [Technical Informational Doc](https://www.notion.so/dydx/44d99ab6a60c4278b9257516b6b75209?v=e14b73b5416f4a6780d44d53b54511dc&pvs=4) and add a link here perhaps

Reply:
added examples

https://www.notion.so/dydx/Metrics-Guidelines-f31f02e792bc4914abac2309df02e3b0?pvs=4
started a doc with a todo here, will work on it later since its not P0